### PR TITLE
BB2-231 Update references to Blue Button on the static website

### PR DESCRIPTION
**Pull request by johnfrenchxyz**
**Context:**
Because there is an original "Blue Button" service where users can download a flat file of their claims data, we want to be sure all the copy on our static site to distinguishes the Blue Button 2.0 API from the original Blue Button service.

This PR has multiple commits of a lot of changes. I `"grepped"` several different ways to find all references. We essentially wanted to take any reference to `Blue Button` and ensure it said either `Blue Button 2.0` or `Blue Button 2.0 API`.

### CODE REVIEW

File: _pages/developers.md [hunk 1]
```diff
@@ -188,9 +188,9 @@ you can do the following:
 
 ## Authorization
 
-To use the Blue Button OAuth 2 a developer must [register their application](https://sandbox.bluebutton.cms.gov/v1/o/applications/).
+To use the Blue Button 2.0 OAuth 2 a developer must [register their application](https://sandbox.bluebutton.cms.gov/v1/o/applications/).
```
Comment (njdister) [inline _pages/developers.md:191]: This has nothing to do with your edit but OAuth 2 is a spec not a feature, so this sentence is somewhat confusing.  I think you could just replace `OAuth 2` with `API`.  Not sure if that needs client approval.

File: _posts/2018-06-07-Bluebutton-FHIR-Implementation-Guide.md [hunk 2]
```diff
@@ -57,15 +57,15 @@ this link: [Blue Button 2.0 Implementation Guide](/assets/ig/index.html).
...
+<li><a href="/assets/ig/StructureDefinition-bluebutton-patient-claim.html">Blue Button 2.0 Patient Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-carrier-claim.html">Blue Button 2.0 PCarrier Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-dme-claim.html">Blue Button 2.0 PDME Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-hha-claim.html">Blue Button 2.0 PHHA Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-hospice-claim.html">Blue Button 2.0 PHospice Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-inpatient-claim.html">Blue Button 2.0 PInpatient Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-outpatient-claim.html">Blue Button 2.0 POutpatient Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-pde-claim.html">Blue Button 2.0 PPart D Event Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-snf-claim.html">Blue Button 2.0 PSNF Claim Profile</a></li>
```
Comment (njdister) [inline _posts/2018-06-07-Bluebutton-FHIR-Implementation-Guide.md:68]: Looks like there may be a substitute and replace problem here where an extra `P` was injected.

File: _posts/2018-06-15-Working-With-FHIR.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ On June 19th, 2018 the Blue Button 2.0 API is being featured in one of the
 at the HL7 [FHIR DevDays conference](https://www.fhirdevdays.com/boston/) in Boston, MA.
 
 The break-out session and the hands-on practice areas feature exercises to work with different aspects of FHIR.
-For Blue Button we are taking developers through interacting with our FHIR API through the OAuth2.0 Interface.
+For Blue Button 2.0 we are taking developers through interacting with our FHIR API through the OAuth2.0 Interface.
```
Comment (njdister) [inline _posts/2018-06-15-Working-With-FHIR.md:32]: Unrelated to your edit, but I would put an space between `OAuth` and `2.0` here.

Comment (johnfrenchxyz) [reply _posts/2018-06-15-Working-With-FHIR.md]: Fixed in commit.

File: _posts/2018-06-07-Bluebutton-FHIR-Implementation-Guide.md [hunk 4]
```diff
@@ -57,15 +57,15 @@ this link: [Blue Button 2.0 Implementation Guide](/assets/ig/index.html).
...
-<li><a href="/assets/ig/StructureDefinition-bluebutton-hha-claim.html">Blue Button HHA Claim Profile</a></li>
-<li><a href="/assets/ig/StructureDefinition-bluebutton-hospice-claim.html">Blue Button Hospice Claim Profile</a></li>
-<li><a href="/assets/ig/StructureDefinition-bluebutton-inpatient-claim.html">Blue Button Inpatient Claim Profile</a></li>
-<li><a href="/assets/ig/StructureDefinition-bluebutton-outpatient-claim.html">Blue Button Outpatient Claim Profile</a></li>
-<li><a href="/assets/ig/StructureDefinition-bluebutton-pde-claim.html">Blue Button Part D Event Profile</a></li>
-<li><a href="/assets/ig/StructureDefinition-bluebutton-snf-claim.html">Blue Button SNF Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-patient-claim.html">Blue Button 2.0 Patient Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-carrier-claim.html">Blue Button 2.0 PCarrier Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-dme-claim.html">Blue Button 2.0 PDME Claim Profile</a></li>
+<li><a href="/assets/ig/StructureDefinition-bluebutton-hha-claim.html">Blue Button 2.0 PHHA Claim Profile</a></li>
```
Comment (njdister) [inline _posts/2018-06-07-Bluebutton-FHIR-Implementation-Guide.md:63]: Looks like these 3 lines still have the `P` prefix issue.

**Review by njdister — APPROVED** [review]
🎉

**Pull request merged**